src/renderer/dx/DxRenderer.cpp

```diff
@@ -12,6 +12,177 @@
 #include "../../inc/DefaultSettings.h"
 #include <VersionHelpers.h>
 
+#include <DirectXMath.h>
+#include <d3dcompiler.h>
+#include <DirectXColors.h>
+
+#pragma comment(lib, "d3dcompiler.lib")

```

Comment:
We'll need to move this into the actual linking list. Also, I'll need to see if it will wreck havoc on the Windows build (and we may need to tweak the linking slightly before commit).